en/developer_guide/multi_device.rst

```diff
@@ -1,24 +1,31 @@
 Multi-device
 ============
 
+Open Orchestra is compatible with the adaptive way of rendering web pages. The idea is to serve
+different html versions of the same document according to the device displaying it. For instance
+an iphone version could show less blocks than a comptuter version.
+
+The main steps of the process are the following:
```

Comment:
there is a format error, I think that you need to let an empty line before the listing